src/full.js

```diff
@@ -20,6 +21,11 @@ const install = function(vue, opts = {}) {
   });
 };
 
+Vue.use(Segment, {
```

Comment:
you can't use vue instance that is a parameter of the install method?

Reply:
Nope last time i tried it doesn't worked :/

Reply:
then it's perfect 👍